docusaurus/docs/recipes/favicon-os-theme.md

```diff
@@ -0,0 +1,35 @@
+---
+id: favicon-os-theme
+title: Changing favicon according to OS theme
+sidebar_label: Changing favicon according to OS theme
+---
+
+If you want to add a different favicon for a specific OS theme you should do the following:
```

Comment:
Please add a brief phrase explaining why we need this. It can be similar to the one you added in the instructions document.